MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -8,9 +8,11 @@ protocol FeedViewControllerDelegate {
 	func didRequestFeedRefresh()
 }
 
-public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView {
+public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView, FeedErrorView {
 	var delegate: FeedViewControllerDelegate?
 
+	@IBOutlet private weak var errorView: ErrorView!
```

Comment:
This property doesn't need to be `weak`.

```suggestion
	@IBOutlet private var errorView: ErrorView!
```